tests/pandas/test_pandas.py

```diff
@@ -714,7 +714,7 @@ def check_invalid_schema_values():
     ):
         client.encrypt_from_pandas(pandas_df, schema=schema_float_column)
 
-    schema_float_oob = {f"{feat_name}_float_1": {"min": float_min // 2, "max": float_max // 2}}
```

Comment:
I'm fine with that but could you maybe explain a little more what's happening here? Having min=max seems kinda weird.

Reply:
we want to always trigger then catch e error, so obviously we need weird configs like this 😅

Reply:
added a comment